opencga-server/src/main/java/org/opencb/opencga/server/rest/JobWSServer.java

```diff
@@ -365,40 +366,31 @@ public Response updateAcl(
         }
     }
 
-
-//    @GET
-//    @Path("/aggregationStats")
-//    @ApiOperation(value = "Fetch catalog job stats", response = FacetField.class)
-//    public Response getAggregationStats(
-//            @ApiParam(value = ParamConstants.STUDY_DESCRIPTION) @QueryParam(ParamConstants.STUDY_PARAM) String studyStr,
-//            @ApiParam(value = "Tool id") @QueryParam("toolId") String toolId,
-//            @ApiParam(value = "Tool scope") @QueryParam("toolScope") String toolScope,
-//            @ApiParam(value = "Tool type") @QueryParam("toolType") String toolType,
-//            @ApiParam(value = "Tool resource") @QueryParam("toolResource") String toolResource,
-//            @ApiParam(value = "User id") @QueryParam("userId") String userId,
-//            @ApiParam(value = "Priority") @QueryParam("priority") String priority,
-//            @ApiParam(value = "Tags") @QueryParam("tags") String tags,
-//            @ApiParam(value = "Executor id") @QueryParam("executorId") String executorId,
-//            @ApiParam(value = "Executor framework") @QueryParam("executorFramework") String executorFramework,
-//            @ApiParam(value = "Creation year") @QueryParam("creationYear") String creationYear,
-//            @ApiParam(value = "Creation month (JANUARY, FEBRUARY...)") @QueryParam("creationMonth") String creationMonth,
-//            @ApiParam(value = "Creation day") @QueryParam("creationDay") String creationDay,
-//            @ApiParam(value = "Creation day of week (MONDAY, TUESDAY...)") @QueryParam("creationDayOfWeek") String creationDayOfWeek,
-//            @ApiParam(value = "Status") @QueryParam("status") String status,
-//            @ApiParam(value = "Release") @QueryParam("release") String release,
-//            @ApiParam(value = "Calculate default stats", defaultValue = "false") @QueryParam("default") boolean defaultStats,
-//            @ApiParam(value = "List of fields separated by semicolons, e.g.: studies;type. For nested fields use >>, e.g.: studies>>biotype;type;numSamples[0..10]:1") @QueryParam("field") String facet) {
-//        try {
-//            query.remove(ParamConstants.STUDY_PARAM);
-//            query.remove("field");
-//
-//            queryOptions.put(QueryOptions.FACET, facet);
-//
-//            DataResult<FacetField> queryResult = catalogManager.getJobManager().facet(studyStr, query, queryOptions, defaultStats,
-//                    token);
-//            return createOkResponse(queryResult);
-//        } catch (Exception e) {
-//            return createErrorResponse(e);
-//        }
-//    }
+    @GET
+    @Path("/aggregationStats")
+    @ApiOperation(value = "Fetch catalog job stats", response = FacetField.class)
+    public Response getAggregationStats(
+            @ApiParam(value = ParamConstants.STUDY_DESCRIPTION) @QueryParam(ParamConstants.STUDY_PARAM) String studyStr,
```

Comment:
Same here. We should add the fields we currently have in the /search and /distinct web services